backend/utils/llm.py

```diff
@@ -1,12 +1,32 @@
 import os
 from dotenv import load_dotenv
 from mistralai.client import MistralClient
+from mistralai.models.chat_completion import ChatMessage
 
 load_dotenv()
 
-endpoint = os.environ["INFER_GPT_MISTRAL_LARGE_ENDPOINT"]
-api_key = os.environ["INFER_GPT_MISTRAL_LARGE_KEY"]
-model = "azureai"
+try:
+  endpoint = os.environ["INFER_GPT_MISTRAL_LARGE_ENDPOINT"]
+  api_key = os.environ["INFER_GPT_MISTRAL_LARGE_KEY"]
+  key = os.getenv("OPENAI_API_KEY")
+  model = "azureai" # might me mistral-large
```

Comment:
In our case this is `azureai`, but as Danny mentioned on a previous PR, this may be different. Maybe should be changed to an environment variable.

Reply:
Changed